ui/perfherder/constants.js

```diff
@@ -107,3 +107,12 @@ export const compareDefaultTimeRange = {
   value: 86400 * 2,
   text: 'Last 2 days',
 };
+
+export const performanceTagsTips = {
+  harness:
+    'patches that updated that harness and caused improvements/regressions',
+  infra:
+    'improvements/regressions caused by infra changes (changes not related to repository code)',
```

Comment:
Could we title case all these tool tip messages?

Reply:
Also, I think we could use the singular for `patches`. Because an alert summaries has a single culprit patch.